src/Uno.Templates/content/unoapp/.globalconfig

```diff
@@ -0,0 +1,4 @@
+is_global = true
+
+# IDE0055: Fix formatting
+dotnet_diagnostic.IDE0055.severity = warning
```

Comment:
I'd rather not add this file. You should be able to set the diagnostic in the .editorconfig

Reply:
.globalconfig is now considered the "modern"/"encouraged" alternative of .editorconfig. I'd definitely use .globalconfigs for setting severities instead of .editorconfigs. It also has a "slight" performance improvement on Roslyn side since globalconfigs are evaluated per project rather than per file.

Reply:
Unless we have something compelling from Microsoft for why we should move away from .editorconfig I'm hesitant to allow yet another file in. Not to mention what you are doing here is already supported in .editorconfig, and we already configure diagnostics in the .editorconfig. We don't need to make it harder on developers by giving them 2 files to maintain.